<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodaySales</class>
 <widget class="QDialog" name="TodaySales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>431</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>당일매출정보</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="t_sales_info_table"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>총 주문건수 :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="tot_order_cnt_label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>판매건수 : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="t_sales_cnt_label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>취소건수 :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="t_cancel_cnt_label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>미처리건수 : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="not_sales_label">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>총 매출 :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="t_tot_sales_label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>1000000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>카드매출 :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="t_card_sales_label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>50000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>현금매출 :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="t_cash_sales_label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>50000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="checkBtn">
      <property name="text">
       <string>확인</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
